LibCarla/source/carla/road/element/Waypoint.h

```diff
@@ -37,7 +37,7 @@ namespace std {
 
     using argument_type = carla::road::element::Waypoint;
 
-    using result_type = uint64_t;
+    using result_type = unsigned long;
```

Comment:
Gate based on OS?

Reply:
Is there any problem using `uint64_t` in Mac?